app/controllers/sessions_controller.rb

```diff
@@ -9,8 +9,11 @@ def new
   def create
     user = login(params[:email], params[:password], params[:remember_me])
     if user
+      token = Jwt::TokenProvider.(user_id: user.id)
```

Comment:
why you do it in sessions, instead of api/v1/sessions. All json routes for SPA apps (like our react app),or for mobile apps, must use api/v1/ prefix. Only monolith rails app will use routes without namespaces.